packages/@cdktf/hcl2json/test/hcl2json.test.ts

```diff
@@ -93,19 +93,15 @@ describe("convertFiles", () => {
     });
   });
 
-  test("invalid files", async () => {
-    try {
-      await convertFiles(path.join(__dirname, "fixtures", "invalid-files"));
-    } catch (e) {
-      expect(e.message).toMatch(/Invalid multi-line string/);
-    }
+  test("invalid files", () => {
+    expect(
```

Comment:
(Blocker)
Mutahhir just recently fixed some tests by returning the result from `expect(...).rejects|resolves` – I think we need to do this here, too.

Related docs: https://jestjs.io/docs/asynchronous#resolves--rejects